src/algorithms/slp_line_search.jl

```diff
@@ -135,6 +154,20 @@ function active_set_optimize!(slp::SlpLS)
 		@printf("  %.8e", sparsity_val)
 		@printf("\n")
         end
+        
+        if slp.options.StatisticsFlag != 0
+	    	append!(statistics["f(x)"],slp.f)
+	    	append!(statistics["ϕ(x_k))"],slp.phi)
+	    	append!(statistics["D(ϕ,p)"],slp.directional_derivative)
+	    	append!(statistics["|p|"],norm(slp.p))
+	    	append!(statistics["μ_merit"],slp.mu_merit)
+	    	append!(statistics["μ_lp"],slp.mu_lp)
+	    	append!(statistics["m(p)"],infeasibility)
+	    	append!(statistics["inf_pr"],prim_infeas)
+	    	append!(statistics["inf_du"],dual_infeas)
+	    	append!(statistics["compl"],compl)
+	    	append!(statistics["Sparsity"],sparsity_val)
```

Comment:
Use `push!` as is recommended to add individual items.

Reply:
#106 created